id,module,name,file,doc,markers,status,message,duration
functional_correctness_test_folder/csn/gpt-5/no_docstr/test_snippet_157.py::test_case_0,functional_correctness_test_folder.csn.gpt-5.no_docstr.test_snippet_157,test_case_0,functional_correctness_test_folder/csn/gpt-5/no_docstr/test_snippet_157.py,,xfail,xfailed,,8.949998300522566e-05
functional_correctness_test_folder/csn/gpt-5/no_docstr/test_snippet_157.py::test_case_1,functional_correctness_test_folder.csn.gpt-5.no_docstr.test_snippet_157,test_case_1,functional_correctness_test_folder/csn/gpt-5/no_docstr/test_snippet_157.py,,xfail,xfailed,,7.866599480621517e-05
functional_correctness_test_folder/csn/gpt-5/no_docstr/test_snippet_157.py::test_case_2,functional_correctness_test_folder.csn.gpt-5.no_docstr.test_snippet_157,test_case_2,functional_correctness_test_folder/csn/gpt-5/no_docstr/test_snippet_157.py,,,failed,AttributeError: type object 'HashMixin' has no attribute 'encrypt_sql'. Did you mean: 'get_encrypt_sql'?,6.766599835827947e-05
functional_correctness_test_folder/csn/gpt-5/no_docstr/test_snippet_157.py::test_case_3,functional_correctness_test_folder.csn.gpt-5.no_docstr.test_snippet_157,test_case_3,functional_correctness_test_folder/csn/gpt-5/no_docstr/test_snippet_157.py,,,failed,AttributeError: type object 'HashMixin' has no attribute 'encrypt_sql'. Did you mean: 'get_encrypt_sql'?,0.0019387079810258001
